dnas/deepkey/sdk/Cargo.toml

```diff
@@ -13,9 +13,15 @@ name = "deepkey_sdk"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-# hc_deepkey_types = { path = "../types" }
-hc_deepkey_types = "0.3"
-hdk = { version = "0.3.0-beta-dev.33" }
+hc_deepkey_types = { workspace = true }
+hdk = { workspace = true }
```

Comment:
Cargo doesn't let me publish with workspace references.  No need to revert though because I have to edit it when publishing.